app-misc/rpick/rpick-0.8.5.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 2019-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Could go EAPI-8 if you want, primary thing of note is that it'll depend on `>=virtual/rust-1.53` if do so but that got stabilized today.